client/src/modules/stock/inventories/registry.js

```diff
@@ -46,6 +46,7 @@ function StockInventoriesController(
     displayName      : 'STOCK.QUANTITY',
     headerCellFilter : 'translate',
     cellClass        : 'text-right',
+    type : 'number',
```

Comment:
Need to add here: 

   enableSorting    : true,

Without it, specifying "type: 'number'" sorts alphabetically/lexically, not numerically (it seems to ignore the type spec).  When you add this, the sorting by quantity numerically works correctly.   This probably also applies to the other items that should be sorted numerically.

Reply:
Are you sure?  I seem to be getting the correct sorting behavior:

![TBPKGKGfoE](https://user-images.githubusercontent.com/896472/100971366-0cd9b600-3537-11eb-862e-4e49133d58b4.gif)

Tested on both Chrome and FF.

Reply:
Tested this again and it works without my fix.  Not sure what happened yesterday!

Reply:
Probably just a mix up of branches 😆